lxd/daemon.go

```diff
@@ -36,6 +39,11 @@ import (
 	"github.com/lxc/lxd/shared/version"
 
 	log "gopkg.in/inconshreveable/log15.v2"
+
+	"github.com/juju/idmclient"
+
+	"gopkg.in/macaroon-bakery.v2-unstable/bakery"
+	"gopkg.in/macaroon-bakery.v2-unstable/httpbakery"
```

Comment:
Those should be moved to the first block.

The way we do imports is:
 - Built-in
 - External
 - Internal
 - Named